content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -25,53 +25,52 @@ You should see two, the first being [Docker Hub](https://hub.docker.com/), but w
   "userId": null
 }
 ```
 
 The unique ID for the built-in registry is always `2`.
 
-Now that we have that registry ID, we can use it to add our Docker image to its catalog. We'll provide `yourname/moving-average:v1` as the x86-image of our microservice:
+Now that we have that registry ID, we can use it to add our Docker image to its catalog. We'll provide `iofog-tutorial/moving-average:v1` as the x86-image of our microservice:
 
-```sh
-# registry ID 2 is the internal private one provided by ioFog
-iofog-controller catalog add \
-  --name "Moving Average" \
-  --x86-image yourname/moving-average:v1 \
-  --registry-id 2 \
-  --user-id 1
+```bash
+$ docker exec -it iofog-controller \
+    iofog-controller catalog add \
+    --name "Moving Average" \
+    --x86-image lkrcal/moving-average:v1 \
+    --registry-id 2 \
+    --user-id 1
 ```
 
-This command will return a catalog ID that we'll use in the next step.
+This command will return a catalog ID that we'll use in the next step, in this case it is `105`.
 
 ## Add Your Microservice
 
 Now that the Docker image containing our microservice code is registered, we can spin up new copies of it also using the Controller.
 
 Instantiating a new microservice is done using the `microservice add` command. We need to provide several options, the most notable being the catalog ID we received in the previous section as well as a node UUID—which is the UUID of the edge node we want this microservice to run on.
 
-So let's find the UUID for the first Agent with the name "Agent 1":
+So let's find the UUID for the first Agent with the name "Default Fog":
 
-```sh
-iofog-controller iofog list
+```bash
+docker exec -ti iofog-controller iofog-controller iofog list
 ```
 
-Using that UUID, we can pass it and our other arguments to instantiate the microservice:
-
-```sh
-iofog-controller microservice add \
-  --name "Moving Average 1" \
-  --catalog-id <catalog_id> \
-  --config '{ "maxWindowSize": 10 }' \
-  --iofog-uuid <iofog_uuid> \
-  --flow-id 1
+In our case, the default agent has a UUID of `vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC`. Using that UUID together with the reference to the catalog, we can pass it and our other arguments to instantiate the microservice. This is also a great opportunity to include our custom config for our `maxWindowSize`.
+
+```bash
+docker exec -it iofog-controller \
+    iofog-controller microservice add \
+    --name "Moving Average" \
+    --catalog-id 105 \
+    --config '{ "maxWindowSize": 10 }' \
+    --iofog-uuid vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC \
+    --flow-id 1 \
+    --user-id 1
 ```
 
-This is also a great opportunity to include our custom config for our `maxWindowSize`.
-
-This command will return the microservice UUID, which we'll then use in the next step to setup our routes.
+This command will return the microservice UUID, which we'll then use in the next step to setup our routes. In our case, the returned UUID of hte microservice is `ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23`.
```

Comment:
Note: _Setup_ is a noun. _Set up_ is a verb.

...to **set up** our routes. ...the returned UUID of **the** microservice...